public/video-ui/src/actions/VideoActions/videoUsages.js

```diff
@@ -0,0 +1,40 @@
+import VideosApi from '../../services/VideosApi';
+
+function requestVideoUsages(id) {
+  return {
+    type:       'VIDEO_USAGE_GET_REQUEST',
+    receivedAt: Date.now()
+  };
+}
+
+function receiveVideoUsages(usages, id) {
+  const usageObject = {};
```

Comment:
this can actually be written as 
`const usageObject = {[id]: usages.response.results}`